demo-server.html

```diff
@@ -5,438 +5,6 @@
 description: Details on how to request a free account on our demo server to enable you to explore the OMERO platform using examples of your own data.
 demo-version:
     - yes
+redirect_to:
+  - https://www.openmicroscopy.org/explore/
```

Comment:
I have asked elsewhere but where is the new reference place for the demo server SLA?

Reply:
Is that http://qa.openmicroscopy.org.uk/registry/demo_account/ ?  This is what is linked from under the blue button on https://www.openmicroscopy.org/explore/ - but there is also the in-line link to https://help.openmicroscopy.org/demo-server.html on [that www page](https://www.openmicroscopy.org/explore/ ) which will redirect to the very same page, which is a bit recursive.


Maybe the link in the https://www.openmicroscopy.org/explore/  page should point to http://qa.openmicroscopy.org.uk/registry/demo_account/  ?
If you agree, I can add a commit to https://github.com/ome/www.openmicroscopy.org/pull/401 if it helps.

Reply:
Ideally this would be moved out of http://qa.openmicroscopy.org.uk/registry/demo_account/ since there's no link between QA and demo. I presume it's leftover from when the signup form was part of QA.

Reply:
My long-term vote would also be to migrate this content outside qa.openmicroscopy.org.uk (in which case the page above can also be redirected towards /explore).

A new static page under www.openmicroscopy.org would be an option. The alternative would be e.g. https://demo.openmicroscopy.org/sla. Once we have settled on the location, the text under https://demo.openmicroscopy.org/signup/ can be updated to link to the SLA

Reply:
It could be added to https://github.com/ome/omero-signup/ either on the signup page or separately, but we'd need to decide is the app specifically for the demo server, or is it for general use in which case the content of the page needs to be configurable, in which case a www page will be quicker to setup.

Reply:
At minimum, I would vote for separating the SLA from the signup page. Ideally, this content should be accessible independently of whether you are creating an account or already an existing user of the service.

Reply:
From a practical point of view, help.o.org/demo-server did not bring you directly to the demo_account. It has a lot of stuff like webadmin, insight etc. i.e. a kind of overview page
https://www.openmicroscopy.org/explore/ is also an "overarching" page with a link to the demo account.

I think redirecting to https://www.openmicroscopy.org/explore/ is more in the spirit of the help.o.org page